### PR TITLE
segmented-control translation, ref #329

### PR DESCRIPTION
**Pull request by silentcloud**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

---
This change is [Reviewable](https://reviewable.io/reviews/ant-design/ant-design-mobile/1441)

### CODE REVIEW

File: components/segmented-control/index.en-US.md [hunk 1]
```diff
@@ -20,12 +19,12 @@ Support WEB, React-Native.
...
-| onChange    |    回调函数     | (e): void |  function(){}  |
-| onValueChange    |    回调函数   | (val): void |  function(){}  |
+| prefixCls(`web only`)  | prefix class        | String |  `am-segment`  |
+| className(`web only`) | class name of component        | String |    |
+| style | style of component        | Object | `{}`   |
+| tintColor  | accent color of the control       | String |  `#2DB7F5`  |
+| disabled  | whether the user can be able to interact with the control  | Boolean |  false  |
+| selectedIndex  | the index in `props.values` of the segment to be (pre)selected | Number |  0  |
+| values  | The labels for the control's segment buttons, in order  | array |  []  |
+| onChange    |  callback that is called when the user taps a segment; passes the event as an argument | (e): void |  function(){}  |
```
Comment (paranoidjk) [inline components/segmented-control/index.en-US.md]: event object

File: components/segmented-control/index.en-US.md [hunk 2]
```diff
@@ -20,12 +19,12 @@ Support WEB, React-Native.
...
-| selectedIndex  | 选中项在数组中的索引        | Number |  0  |
-| values  | 选项数组,值是字符串        | array |  []  |
-| onChange    |    回调函数     | (e): void |  function(){}  |
-| onValueChange    |    回调函数   | (val): void |  function(){}  |
+| prefixCls(`web only`)  | prefix class        | String |  `am-segment`  |
+| className(`web only`) | class name of component        | String |    |
+| style | style of component        | Object | `{}`   |
+| tintColor  | accent color of the control       | String |  `#2DB7F5`  |
+| disabled  | whether the user can be able to interact with the control  | Boolean |  false  |
+| selectedIndex  | the index in `props.values` of the segment to be (pre)selected | Number |  0  |
```
Comment (paranoidjk) [inline components/segmented-control/index.en-US.md]: pre 是啥意思？

File: components/segmented-control/index.en-US.md [hunk 3]
```diff
@@ -20,12 +19,12 @@ Support WEB, React-Native.
...
-| disabled  | 是否启用        | Boolean |  false  |
-| selectedIndex  | 选中项在数组中的索引        | Number |  0  |
-| values  | 选项数组,值是字符串        | array |  []  |
-| onChange    |    回调函数     | (e): void |  function(){}  |
-| onValueChange    |    回调函数   | (val): void |  function(){}  |
+| prefixCls(`web only`)  | prefix class        | String |  `am-segment`  |
+| className(`web only`) | class name of component        | String |    |
+| style | style of component        | Object | `{}`   |
+| tintColor  | accent color of the control       | String |  `#2DB7F5`  |
+| disabled  | whether the user can be able to interact with the control  | Boolean |  false  |
```
Comment (paranoidjk) [inline components/segmented-control/index.en-US.md]: whether the user can  interact with the control 

or

whether the user is  able to  interact with the control

File: components/segmented-control/index.en-US.md [hunk 4]
```diff
@@ -4,14 +4,13 @@ type: Navigation
...
-
-### 规则
-- 和 Tabs 功能相似，尽可能避免一个页面中同时出现这两个组件。
-- 可以搭配 NavBar 一起使用，用于显示多个视图，分段数一般为 2 个。
-- 单独放置一行时，分段数最多为 5 个；文案需要精简，一般 2-4 个字。
-- 尽可能保持文案长度一致。
+### Rule
+- It is similar to the functionality used for `Tabs`, so avoid to use them at same page as much as possible.
+- You can use `SegmentedControl` with `NavBar` to display mutiple views.
+- The number of segments is at most five in one use case. Each segment generally has 2-4 words and needs simplified texts.
```
Comment (paranoidjk) [inline components/segmented-control/index.en-US.md]: Generally there should be no more than 5 segments in one line.

File: components/segmented-control/index.en-US.md (hunk 2 above)
Comment (silentcloud) [inline components/segmented-control/index.en-US.md]: selectedIndex 是从 RN 那边复制过来的

File: components/segmented-control/index.en-US.md (hunk 4 above)
Comment (silentcloud) [inline components/segmented-control/index.en-US.md]: in one line 有点奇怪

**Pull request merged**